scanner/mapping.go

```diff
@@ -128,12 +128,15 @@ func (s mediaFileMapper) trackID(md metadata.Tags) string {
 	return fmt.Sprintf("%x", md5.Sum([]byte(md.FilePath())))
 }
 
-func (s mediaFileMapper) albumID(md metadata.Tags, releaseDate string) string {
+func (s mediaFileMapper) albumID(md metadata.Tags, releaseDate string, mbzAlbumID string) string {
```

Comment:
conflicts with https://github.com/navidrome/navidrome/pull/2524 , if this PR goes through first, then I need to amend that PR, and vice versa.